docs/reference/schema-viewer-navigation.md

```diff
@@ -0,0 +1,179 @@
+
+Alternatively, you can navigate the Schema Browser by interacting directly with the visual representation. Each schema element is denoted by a bullet. By clicking on the blue bullets, you can easily expand the tree structure to reveal the hierarchical relationships. Clicking (1) on the bullet text (e.g. JdbcTableDataObject) triggers the immediate display of the corresponding schema information (2).
+
+![Image of schema viewer information](../images/schema-viewer-information.png)
+
+## Mapping schema to config file
+
+Generally speaking you can take this as a reference, the naming convention for elements in the configuration file generally mirrors the nomenclature of the elements in the schema browser, with some notable omissions. These omissions include the element specifications such as `{}`, `[mapOf]`, `[oneOf]`, `[]`, or `(type)`. Let's delve into the specifics of each element designation.
+
+When an element's name concludes with `{}`, it signifies an object capable of possessing properties and nested elements that require a designated key.
+
+Example from schema browser (schema{}->global{}):
+```
+global {
+  spark-options {
+    ...
+  } # this a the nested element in global
+  synchronousStreamingTriggerIntervalSec = 2 # this is a property of global{}
+}
+
+...
+
+```
+
+Elements denoted by names ending in `[mapOf]` indicate the existence of a map, specifically a key-value pair of distinct sub-elements, with the element itself lacking any properties.
+
+Example from schema browser (schema{}->dataObjects[mapOf]*):
+```
+dataObjects {
+  ext-departures {
+    type = WebserviceFileDataObject
+    url = "https://opensky-network.org/api/flights/departure?airport=LSZB&begin=1696854853&end=1697027653"
+    readTimeoutMs=200000
+  } # first key-value pair
+
+  ext-airports {
+    type = WebserviceFileDataObject
+    url = "https://davidmegginson.github.io/ourairports-data/airports.csv"
+    followRedirects = true
+    readTimeoutMs=200000
+  } # second key-value pair
+}
+
+...
+
+```
+It is worth noting that the `WebserviceFileDataObject` element appears twice, each time with distinct keys. Initially, it is employed for downloading departure dates, and subsequently for acquiring airport information data.
+
+In contrast to `[mapOf]` elements, elements marked by `[oneOf]` can only accommodate a single nested element. These elements function as a specialized type of property, demanding the inclusion of one specific sub-element.
+
+Example from schema browser (schema{}->actions[mapOf]*->CopyAction{}->executionMode[oneOf]):
+
+:::info 
+There is a special property called (const). It is found on the type property of an object.  This particular property is required to contain the class name of the parent element e.g. CopyAction. 
```

Comment:
This description is good. Maybe it should be mentioned that for elements having the type property, infront to the block the custom/user defined name of the object is defined, also called the objectID.